specification/purview/Azure.Analytics.Purview.DataMap/client.tsp

```diff
@@ -0,0 +1,138 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using PurviewDataMap;
+
+@TypeSpec.Versioning.useDependency(PurviewDataMap.ApiVersions.v2023_09_01)
+namespace Customizations;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@client({
+  name: "DataMapClient",
+  service: PurviewDataMap,
+})
+@operationGroup
```

Comment:
According to https://azure.github.io/typespec-azure/docs/howtos/DataPlane%20Generation%20-%20DPG/02client#one-client-and-two-operation-groups, `@client` part shall be put for namespace.